src/components/About/AboutEnabledModules.js

```diff
@@ -33,7 +33,8 @@ class AboutEnabledModules extends React.Component {
 
       return (
         <li key={key} style={style}>
-          {this.props.availableModules[key]} (<tt>{key}</tt>)
+          {this.props.availableModules[key]}
+          <tt>{`(${key})`}</tt>
```

Comment:
Only concern here is does this cause a visual regression if `tt` is has alternate font styles? If there are no differences in styling, then ignore this

Reply:
It will cause a visual change to the parentheses, but not one that matters.